documentation/src/main/asciidoc/topics/contributing/ui.adoc

```diff
@@ -0,0 +1,56 @@
+[[design_guidelines]]
+= UI Writing Guidelines
+
+This section describes guidelines for working with text in {brandname} user
+interface (UI) elements and applies, but is not limited, to the following
+components:
+
+* {brandname} Console
+* {brandname} CLI
+* log messages
+* error messages
+
+Content in the {brandname} UI should adhere to the link:https://www.patternfly.org/v4/design-guidelines/content/about[PatternFly content style guide].
+
+While the PatternFly design guidelines are the definitive resource for UI writing guidelines, here are some rules for creating microcopy in {brandname} UI:
+
+* Headings on UI elements use sentence casing. link:https://www.patternfly.org/v4/ux-writing/capitalization/[PatternFly capitalization rules]
+* Dialogs that prompt for user action should have the following structure:
++
+Heading in question form, starting with verb. ("Create new entry?") +
+Body that provides details and outcome of action. +
+Body that uses a synonym to the verb in heading where ambiguity might exist. +
+Buttons.
+* Use past tense form for success or failure messages.
++
+*Yes:* "Cache created." *No:* "Cache has been created."
+* Avoid using the word "please".
+* Ensure all icons and buttons with action have tooltip/hover help.
+* Avoid abbreviations such as "avg" instead of "average".
+
+.Terminology
+
+* Use "Create" to mean build something new. For example, "create a cache".
+* Use "Add" to describe adding an existing item to an existing list, group, view, or other container element. For example, "add values to counters".
+
+|===
+|Method |Verb
+
+|`cache.put()`
+|Create
```

Comment:
@tristantarrant and @karesti As part of the Console UI review, we've got the recommendation to use "Create entries" instead of "Add entries". I'm not sure if I agree with that though. First you create the cache, then you add entries to the cache. I know it probably seems super trivial but it's important to make this consistent. Please let me know your thoughts. 

Also, I've gone through the REST API and some other docs. Are there any other methods that should have a corresponding verb in UI strings? I think it'd be helpful to have a list of what's in the code vs. how it should be presented via UI. Thanks.